src/src/DataStructs/EventQueue.h

```diff
@@ -23,6 +23,18 @@ struct EventQueueStruct {
 
   bool isEmpty() const;
 
+  std::size_t size() {
+    return _eventQueue.size();
+  }
+
+  std::list<String>::iterator begin() {
+    return _eventQueue.begin();
+  }
```

Comment:
These can be a `const_iterator begin() const` as you're not doing anything with the iterator, nor deleting it from the queue.

Reply:
Removed, as no longer needed